### PR TITLE
1 rename repo and files using hyphens instead of underscores

### LINKED ISSUE #1: Rename repo and files using hyphens instead of underscores
Since the output of this repo is a website with different subpages, I highly recommend using hyphens for files and the repo. The two most most compelling reasons why to use hyphens are:

1. Search engines like Google treat hyphens as word separators, making it easier for them to understand the words in the URL. 
2. Hyphens make URLs more readable for humans because URLs are often underlined, so underscores "disappear". 

I think it is better to make this breaking change as soon as possible before the docs are linked any further. 

I use underscores for all object and function names within a script and try to use hyphens for all files (incl. script files). Files that carry the name of a function are excluded from this rule (e.g. in package development).

### PR DESCRIPTION
**Pull request by oporeuben**

### CODE REVIEW

File: renv.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "R": {
-    "Version": "4.4.1",
+    "Version": "4.3.2",
```
Comment (chriscarrollsmith) [inline renv.lock]: Hi Reuben! The pull request looks great except for this change. It looks like you need to update your R version!

**Review by chriscarrollsmith — CHANGES_REQUESTED** [review]

File: renv.lock (hunk 1 above)
Comment (oporeuben) [reply renv.lock]: Thanks, let me update my R version.

File: .DS_Store
Comment (chriscarrollsmith) [inline .DS_Store:1]: Add ".DS_Store" to the .gitignore file, then run `git rm --cached .DS_Store` from the Terminal.

File: renv.lock (hunk 1 above)
Comment (chriscarrollsmith) [reply renv.lock]: Once you update your R version, you will need to run `renv::snapshot()` to update the lock file, and then stage, commit, and push the changes.

**Review by chriscarrollsmith — APPROVED** [review]

**Pull request merged**